components/form/input.tsx

```diff
@@ -24,6 +24,12 @@ export default function Input({
   hideLabel,
   ...additionalInputProps
 }: InputProps): JSX.Element {
+  const { conditionalDisplay, ...otherValues } = additionalInputProps;
+  const additionalInputPropsWithConditionalDisplay = {
+    ...otherValues,
+    conditionaldisplay: conditionalDisplay,
+  };
+
```

Comment:
What's the difference here?

Let's say `additionalInputProps` is a set with items A, B, C.
We destructure it to: `{ A, theRest }`
Then recreate the object to be a set that's: `{ theRest, A }`, which is just: `{B, C, A}`

`additionalInputProps` and `additionalInputPropsWithConditionalDisplay` have the same keys.

Reply:
Oh, it's just the casing change of `D` to `d`?

Reply:
But then why not pass it to the input as a (lowercased) `conditionaldisplay` from the get go?

Reply:
yeah exactly - it's just to get rid of the warning without cleaning up the whole thing.